### PR TITLE
Update aws.md to include Cluster access

### PR DESCRIPTION
**Pull request by Kartstig**
I was recently using your guide to set up Nextflow, and I ran into some missing roles when it came to setting up AWS Batch Compute Environment. After I added the following roles, I was able to get it working.

### CODE REVIEW

**Comment by pditommaso** [comment]
Not aware of the requirement of those permissions

**Comment by Kartstig** [comment]
@pditommaso admittedly, I could be wrong. My specific scenario was setting up [Fargate & Fargate Spot compute_resources using terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/batch_compute_environment#fargate-type). I wonder if this is necessary with the Batch settings outlined in the docs.

**Comment by pditommaso** [comment]
I see. In that case it could make sense to add into a separate section highlighting as extra permission for Fargate

**Review by pditommaso — APPROVED** [review]

File: docs/aws.md [hunk 1]
```diff
@@ -85,6 +85,15 @@ Minimal permissions policies to be attached to the AWS account used by Nextflow
...
   ```
 
+Note: If you are running Fargate or Fargate Spot, you may need the following policies in addition to the listed above:
+  ```json
+  "ecs:CreateCluster"
+  "ecs:DeleteCluster"
+  "ecs:DescribeClusters"
+  "ecs:ListClusters"
+  "ec2:DescribeSubnets"
+  ```
```
Comment (bentsherman) [inline docs/aws.md:95]: I would make this a note:

```md
:::{note}
If you are running Fargate or Fargate Spot, you may need the following policies in addition to the listed above:
# ...
:::
```

Comment (pditommaso) [reply docs/aws.md:95]: Done c02e58c7

**Pull request merged**